### PR TITLE
[WIP] Xctest parser

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

File: cloud_testing/xctest_parser/build_earlgrey_example.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  -workspace EarlGreyExample.xcworkspace \
+  -scheme "EarlGreyExampleSwiftTests" \
+  -destination "generic/platform=iOS" \
+  -derivedDataPath "$DD_PATH"
+
+# Run
+# xcodebuild \
+#   -xctestrun "$DD_PATH/Build/Products/EarlGreyExampleSwiftTests_iphonesimulator11.3-x86_64.xctestrun" \
+#   -destination "platform=iOS Simulator,name=iPhone 8 Plus,OS=latest" \
+#   test-without-building
```
Comment (Macarse) [inline cloud_testing/xctest_parser/build_earlgrey_example.sh]: remove?

File: cloud_testing/xctest_parser/flank.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+appApk: ../../test_app/apks/app-debug.apk
+testApk: ../../test_app/apks/app-debug-androidTest.apk
+rootGcsBucket: tmp_bucket_2
+autoGoogleLogin: true
+useOrchestrator: true
```
Comment (Macarse) [inline cloud_testing/xctest_parser/flank.yml]: isn't this flag espresso only?

File: cloud_testing/xctest_parser/build_earlgrey_example.sh (hunk 1 above)
Comment (bootstraponline) [reply cloud_testing/xctest_parser/build_earlgrey_example.sh]: Fixed, thanks!

File: cloud_testing/xctest_parser/flank.yml (hunk 2 above)
Comment (bootstraponline) [reply cloud_testing/xctest_parser/flank.yml]: Fixed

**Pull request merged**